lib/util/ImpCentralApiHelper.js

```diff
@@ -220,6 +204,26 @@ class ImpCentralApiHelper {
         });
     }
 
+    _refreshAccessToken() {
+        if (!this._authConfig.refreshToken && !this._authConfig.loginKey) {
+            return Promise.reject(new Errors.AccessTokenExpiredError(this._authConfig));
```

Comment:
Is in this case access token really expired or is it a misconfiguration or something?

Reply:
It is for --temp option of impt auth login.
In this case impt does not save information (refreshToken or loginKey) required to refresh access token in .impt.auth file and there is no possibility to obtain new access token after the previous one has expired.
I added a comment to the source.